xsql/parser.go

```diff
@@ -884,6 +891,9 @@ func validateStream(stmt *StreamStmt) error {
 	case common.FORMAT_JSON:
 		//do nothing
 	case common.FORMAT_BINARY:
+		if stmt.StreamType == TypeTable {
```

Comment:
Would it be better to validate in the runtime instead of lexical phase? It could be probably complex to make this extensible?

Reply:
extensible not supported yet so add the check here for a quick failure